tests/framework/PostProcessors/Validation/test_validation_gate_probabilistic.xml

```diff
@@ -28,20 +28,20 @@
     <PostProcessor name="pp1" subType="Validation">
       <Features>outputDataMC1|ans</Features>
       <Targets>outputDataMC2|ans2</Targets>
-      <!-- <Metric class="Metrics" type="RACDistance">fz_expected</Metric> -->
-      <Metric class="Metrics" type="CDFAreaDifference">cdf_diff</Metric>
-      <Metric class="Metrics" type="PDFCommonArea">pdf_area</Metric>
+      <!--
+        <Metric class="Metrics" type="RACDistance">fz_expected</Metric>
```

Comment:
This line probably need to be removed.

Reply:
The original author placed it there; I don't have a particular reason to modify it as part of this PR.